charts/posthog/values.yaml

```diff
@@ -1,6 +1,12 @@
-# Default values for posthog.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+## @section Helm parameters
```

Comment:
This doesn't work with helm-docs last I checked. What is this formatting?

Reply:
I was thinking to slowly move to https://github.com/bitnami-labs/readme-generator-for-helm as it looks a bit more complete in my opinion (happy to chat if you think we shouldn't do it!)

Reply:
No strong opinions on tools to use and the tool looks good - only thing is that we'll break ALL_VALUES.md quite bad with this change. 

If you're sold on the tool, let's switch over ASAP, but in a separate PR.

Reply:
Ack, reverted (added as well fixes for the Redis and ClickHouse sections I've edited few PRs ago)